### PR TITLE
Update _rpm_common.sh

### PR DESCRIPTION
**Pull request by PatrickHeppler**
Added switch to use either yum or dnf (fedora 22)

### CODE REVIEW

**Comment by kuba** [comment]
It would be easier to maintain this file if you used `yum_or_dnf=...; $yum_or_dnf git-core ...` instead of duplicating packages list.

**Comment by PatrickHeppler** [comment]
@kuba 
Fixed it

File: bootstrap/_rpm_common.sh [hunk 1]
```diff
@@ -5,15 +5,29 @@
...
-  libffi-devel \
-  ca-certificates \
+bootstrap() {
+
+    pkgs="git-core
+        python 
+        python-devel 
+        python-virtualenv 
+        python-devel 
+        gcc dialog
```
Comment (kuba) [inline bootstrap/_rpm_common.sh]: this should be split into 2 lines

File: bootstrap/_rpm_common.sh [hunk 2]
```diff
@@ -5,15 +5,29 @@
 #   - Centos 7 (x64: on AWS EC2 t2.micro, DigitalOcean droplet)
 
 # "git-core" seems to be an alias for "git" in CentOS 7 (yum search fails)
```
Comment (kuba) [inline bootstrap/_rpm_common.sh]: this comment should be moved down, just next to `pkgs`

File: bootstrap/_rpm_common.sh [hunk 3]
```diff
@@ -5,15 +5,29 @@
...
-  python-devel \
-  python-virtualenv \
-  python-devel \
-  gcc \
-  dialog \
-  augeas-libs \
-  openssl-devel \
-  libffi-devel \
-  ca-certificates \
+bootstrap() {
```
Comment (kuba) [inline bootstrap/_rpm_common.sh]: Why are we wrapping this into a function? Seems not in line with `_deb_common.sh`

File: bootstrap/_rpm_common.sh [hunk 4]
```diff
@@ -5,15 +5,29 @@
...
+        python-devel 
+        python-virtualenv 
+        python-devel 
+        gcc dialog 
+        augeas-libs 
+        openssl-devel 
+        libffi-devel 
+        ca-certificates"
+
+    if hash yum 2>/dev/null; then
```
Comment (kuba) [inline bootstrap/_rpm_common.sh]: `type` is more common for this task (I actually had to look up docs on `hash`)

File: bootstrap/_rpm_common.sh [hunk 5]
```diff
@@ -5,15 +5,29 @@
...
+      yum install -y $pkgs;
+
```
Comment (kuba) [inline bootstrap/_rpm_common.sh]: you could reduce spacing here

File: bootstrap/_rpm_common.sh [hunk 6]
```diff
@@ -5,15 +5,29 @@
...
+        python-virtualenv 
+        python-devel 
+        gcc dialog 
+        augeas-libs 
+        openssl-devel 
+        libffi-devel 
+        ca-certificates"
+
+    if hash yum 2>/dev/null; then
+      yum install -y $pkgs;
```
Comment (kuba) [inline bootstrap/_rpm_common.sh]: semicolon is not necessary

**Comment by kuba** [comment]
Generally looks fine, but you could apply small fixes suggested above

**Comment by PatrickHeppler** [comment]
What the fuck is wrong with you? My script works fine. I don't give a shit about if it's "in line" with _deb_common.sh , I don't give a shit about what you think "is a more common task" and I don't give a fuck about spacing. Who do you think you are, to tell me how to code? Now take it or leave it!

**Pull request closed without merging**